chart/values.yaml

```diff
@@ -349,6 +349,8 @@ csi:
       ctrl_loss_tmo: "1980"
       # Kato (keep alive timeout) in seconds
       keep_alive_tmo: ""
+      # Fallback to nvme connect over tcp if host is not rdma capable
+      nvme_connect_fallback: true
```

Comment:
Maybe something like this?

```suggestion
      tcpFallback: true
```

Reply:
> Maybe something like this?

I can change this but we'll have inconsistency then, with other existing fields. wdyt?

Reply:
converted to camelCase for this new value.